mod/calculated_metrics/lib/formula/calculator/input_item/options/not_researched_option/not_researched_pass_value.rb

```diff
@@ -0,0 +1,42 @@
+module Formula
+  class Calculator
+    class InputItem
+      module Options
+        module NotResearchedOption
+          # Used if the "not researched" option is set to an arbitrary return value
+          module NotResearchedPassValue
+            def value_for company_id, year
+              replace_nil super
+            end
+
+            def not_researched_value
+              case not_researched_option
+              when "false" then false
+              when /^[+-]?\d+$/ then not_researched_option.to_i
+              else
+                not_researched_option
+              end
+            end
+
+            def replace_nil value
+              return value unless input_value_not_researched?(value)
+              if value.is_a?(Array)
+                value.map { |v| v.nil? ? not_researched_value : v }
+              else
+                not_researched_value
+              end
+            end
+
+            def mandatory?
+              false
+            end
+
+            def sort_index
+              super + 2 * parser.input_count
+            end
```

Comment:
maybe a comment here?

Reply:
sure (there is a comment where `sort` is called  which gives you at least a chance to understand what this is about)